pywbem/mof_compiler.py

```diff
@@ -599,42 +599,55 @@ def p_mp_createClass(p):
                     objects = list(cc.properties.values())
                     for meth in cc.methods.values():
                         objects += list(meth.parameters.values())
+                    # TODO the test for class in list does not work now that
+                    # we have made dep_classes a tuple with class and ce
```

Comment:
* In the code, `dep_classes` is still a list, not a tuple as stated in the TODO text.
* There's something missing at the end of the TODO text.